### PR TITLE
Replace photo on home page with 2018 worlds photo

### LINKED ISSUE #231: Use high-res team photo from Championship on homepage
Need to get this from whoever was taking photos with a good camera (not Knight).

### PR DESCRIPTION
**Pull request by JosephCarpenter**
Fix #231

### CODE REVIEW

**Comment by JosephCarpenter** [comment]
There seem to be a couple wonky extra commits, most likely due to all the git gymnastics I had to do from my computer reimaging. Should I try to remove them?

**Comment by ErikBoesen** [comment]
Yeah, fix that.

File: index.html [hunk 1]
```diff
@@ -23,16 +23,18 @@
     <h1 id="welcome">Welcome!</h1>
     <!-- TODO: Replace this tagline with something less narcissistic -->
     <h3>We are one of the most successful and inventive FIRST Robotics teams in the Chesapeake District.</h3>
-    <a href="/robot/2018" class="button">View our 2018 Robot</a>
+    <a href="/robot/2017" class="button">View our 2017 Robot</a>
```
Comment (ErikBoesen) [inline index.html]: Yeah this stuff is because of the duplicates.

Comment (JosephCarpenter) [reply index.html]: oof

**Comment by ErikBoesen** [comment]
Also, I was thinking of the photo in front of the FIRST sign, if you can get that from Zack.

**Comment by JosephCarpenter** [comment]
So was I, but that was taken by Mr. Knight, presumably from his phone camera. Should I still try to get that from him?

**Comment by ErikBoesen** [comment]
Yeah, definitely.

**Comment by ErikBoesen** [comment]
I don't think we need this anymore, do we?

**Pull request closed without merging**